lib_term/analysis.ml

```diff
@@ -114,6 +114,7 @@ module Make (Meta : sig type t end) = struct
   end
 
   let colour_of_activity = function
+    | `Waiting_for_confirmation -> "#4975ff"
```

Comment:
I was not sure what colour to choose for this so to signify some sort of a stopped state I chose "blue"
![scrn-2021-05-24-01-42-12](https://user-images.githubusercontent.com/2611789/119282395-7064b680-bc31-11eb-8b76-4cb56eaa6a5f.png)